client/src/styles/Card.css

```
@@ -0,0 +1,20 @@
+.card {
+    /* border: solid rgb(133, 131, 131) 5px; */
+    box-shadow: 0px 10px 10px rgb(114, 112, 112);  
+    max-height: 400px;
+}
+
+.card-header .title {
+    font-size: 30px;
+    margin: 2px;
+}
+
+.card-header {
+    display: flex;
+    flex-direction: column;
+}
+
+.card-header .status-icon{
+    font-size: 40px;
+    align-self: end;
+}
```

I'd suggest to reorder these so it will be easier to see the structure of your css classes.
```suggestion
.card-header {
    display: flex;
    flex-direction: column;
}

.card-header .title {
    font-size: 30px;
    margin: 2px;
}

.card-header .status-icon{
    font-size: 40px;
    align-self: end;
}
```